packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1170,6 +1180,31 @@ describe('network stubbing', { retries: 2 }, function () {
           })
         })
       })
+
+      context('with `times`', function () {
+        it('only uses each handler N times', function () {
+          const third = sinon.stub()
+
+          cy.intercept({ url: '/foo', times: 1 }, 'first')
+          .intercept({ url: '/foo', times: 2 }, 'second')
+          .intercept({ url: '/foo', times: 2 }, third)
+          .intercept({ url: '/foo', times: 3 }, 'fourth')
+          .then(async () => {
+            const expectGet = (expected) => $.get('/foo').then((res) => expect(res).to.eq(expected))
+
+            await Promise.mapSeries([
+              'first',
+              'second',
+              'second',
+              'fourth',
+              'fourth',
+              'fourth',
+            ], expectGet)
+
+            expect(third).to.be.calledTwice
```

Comment:
can we assert that after the intercepts are exhausted they no longer apply?

Reply:
that is what this test is doing

Reply:
do you mean like via cy.wait?

Reply:
I mean more like that the very last intercept `.intercept({ url: '/foo', times: 3 }, 'fourth')` is not longer used after 3 times, and the endpoint receives it.

Reply:
alright, i've added a check that a final request falls through, as well as asserting that `alias.all` matches the expected length